data/test/scenarios/poison_opponent.cfg

```diff
@@ -1,4 +1,16 @@
 # this unit test what 'poison' applied to opponent work perfectly after fixing.
+
+#####
+# API(s) being tested: [poison]apply_to=opponent
+##
+# Actions:
+# Give both units the poison weapon special.
+# Have both units hit each other.
+##
+# Expected end state:
+# Both units are poisoned.
+#####
+
```

Comment:
A general note - it seems inconsistent whether there's a blank line between the comment and the test or not. I'd prefer to omit this blank line from all the tests).

Reply:
Updated.